src/js/providers/html5.js

```diff
@@ -2,7 +2,7 @@ import { qualityLevel } from 'providers/data-normalizer';
 import { Browser, OS } from 'environment/environment';
 import { isAndroidHls } from 'providers/html5-android-hls';
 import { STATE_IDLE, STATE_PLAYING, MEDIA_META, MEDIA_ERROR,
-    MEDIA_LEVELS, MEDIA_LEVEL_CHANGED, MEDIA_SEEK } from 'events/events';
+    MEDIA_LEVELS, MEDIA_LEVEL_CHANGED, MEDIA_SEEK, STATE_BUFFERING } from 'events/events';
```

Comment:
Remove `STATE_BUFFERING` and use `STATE_LOADING` (or stalled for under-buffering).

Reply:
👍